daemon/mgr/image.go

```diff
@@ -211,6 +189,16 @@ func (mgr *ImageManager) RemoveImage(ctx context.Context, idOrRef string, force
 		return err
 	}
 
+	// since there is no rollback functionality, no guarantee that the
+	// containerd.RemoveImage must success. so if the localStore has been
+	// remove all the primary references, we should clear the CtrdImageInfo
+	// cache.
+	defer func() {
```

Comment:
Whether we should remove the cache image info when the image has been succeeded?